ArkEcosystem/explorer
# refactor: add dark theme script

## Summary
https://app.clickup.com/t/1xw86by
~~waiting on https://github.com/ArkEcosystem/laravel-foundation/pull/200~~

This PR adds the Dark Theme script and adjust the current implementation.

## How to test
- open the homepage
- go to the menu (top/right) and toggle dark mode
- check if every component in the page uses the correct color (font, background, border, etc)
- repeat by opening different pages (don't forget to test charts on `/statistics` page too)

## Checklist
